File: app/src/main/java/com/voiceassist/lixinyu/voiceassist/common/widget/grid/BaseGridAdapter.java
package com.voiceassist.lixinyu.voiceassist.common.widget.grid;

import android.view.View;

/**
 * Created by lilidan on 2018/1/22.
 */

public interface BaseGridAdapter {

    int getColumCount();
    int getCount();
    View getView(int position);
}

File: app/src/main/java/com/voiceassist/lixinyu/voiceassist/common/widget/grid/BaseGridAdapter.kt
package com.voiceassist.lixinyu.voiceassist.common.widget.grid

import android.view.View

/**
 * Created by lilidan on 2018/1/22.
 */

interface BaseGridAdapter {

    val columCount: Int
    val count: Int
    fun getView(position: Int): View
}
